tools.py

```
@@ -0,0 +1,66 @@
+from datetime import datetime
```

Ruff (0.8.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)